src/main/webapp/js/script.js

```diff
@@ -76,3 +76,58 @@ function showOrHideNotes(version) {
     $('div.setup-panel div a.btn-success').trigger('click');
 });
 
+// Enroll Form Validation using bootstrapValidator Framework
+
+$(document).ready(function(){
+    $(".enrollForm").bootstrapValidator({
+
+        message: "This value is not valid",
+        feedbackIcons: {
+                valid: 'glyphicon glyphicon-ok',
+                invalid: 'glyphicon glyphicon-remove',
+                validating: 'glyphicon glyphicon-refresh'
+            },
+
+        // Validation of fields form
+        fields: {
+            currentVersion: {
+              message: 'The username is not valid',
+                validators: {
+                    notEmpty: {
+                        message: 'The current version is required and cannot be empty'
+                    },
+                    stringLength: {
+                        min: 6,
+                        max: 6,
+                        message: 'The current version must be 6 characters long'
```

Comment:
What about 19.10.1? I don't think we need this check. Did you see such restrictions on version numbers somewhere?